### PR TITLE
[#8] Search changed from exact to LIKE

### PR DESCRIPTION
**Pull request by eddiejaoude**

### CODE REVIEW

File: src/main/java/com/empowerhack/family/memberservice/Repository/MemberRepository.java [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 import java.util.List;
 import com.empowerhack.family.memberservice.Entity.Member;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.data.repository.query.Param;
 
 public interface MemberRepository extends PagingAndSortingRepository<Member, Long> {
 
+    @Query("SELECT m FROM Member m WHERE m.name LIKE %:name%")
```
Comment (brettminnie) [inline src/main/java/com/empowerhack/family/memberservice/Repository/MemberRepository.java:11]: Do you not need to escape the literal in single quotes? Would that not render to %'(value)'%

Comment (eddiejaoude) [inline src/main/java/com/empowerhack/family/memberservice/Repository/MemberRepository.java:11]: Apparently not, thats what I did first, but it didn't work, then reading the documentation, this is the correct way to achieve the **LIKE** query http://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.at-query

Comment (brettminnie) [inline src/main/java/com/empowerhack/family/memberservice/Repository/MemberRepository.java:11]: More magic eh?

Comment (eddiejaoude) [inline src/main/java/com/empowerhack/family/memberservice/Repository/MemberRepository.java:11]: Convention over configuration is the way I look at it :wink:

**Pull request merged**